docs-v2/integrations/all/harvest/connect.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Harvest - How do I link my account?
+sidebarTitle: Harvest
+---
+
+
+# Overview
+
+To authenticate with Harvest, you need:
+1. **App Details** - A unique identifier for your integration, typically included in the `User-Agent` header when sending API requests. It provides information about your application, such as its name and a point of contact, to enable communication with Harvest if needed.
+
+This guide will walk you through creating **App Details** for Harvest API integration.
+
+### Prerequisites:
+
+- You must have an account with Harvest.
+
+### Instructions:
+
+#### Step 1: Creating App Details
+1. Select a unique and descriptive name that represents your application or integration. For example, **Nango** or **Nango's Harvest Integration**.
+2. Include a way for Harvest to contact you. This can be either:
+- A URL to your application or support page, such as `http://nango.com/contact`.
+- An email address for direct communication, such as `support@nango.com`.
```

Comment:
not really advisable to ask customer to redirect support to us

Reply:
I was using nango as the example company 😅 , updated with the latest commit.